packages/mobile/src/screens/chat-screen/ChatUser.tsx

```diff
@@ -0,0 +1,60 @@
+import type { User } from '@audius/common'
+import { View } from 'react-native'
+
+import { Text } from 'app/components/core'
+import { ProfilePicture } from 'app/components/user'
+import { UserBadges } from 'app/components/user-badges'
+import { makeStyles } from 'app/styles'
+
+const useStyles = makeStyles(({ spacing, palette, typography }) => ({
+  root: {
+    display: 'flex',
+    flexDirection: 'row'
+  },
+  profilePicture: {
+    width: spacing(12),
+    height: spacing(12),
+    borderWidth: 1,
+    borderColor: palette.neutralLight9
+  },
+  userContainer: {
+    display: 'flex',
+    flexDirection: 'column',
+    paddingTop: 2,
+    marginLeft: spacing(2),
+    marginBottom: spacing(2)
+  },
+  userNameContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    marginBottom: spacing(1)
+  },
+  userName: {
+    fontSize: typography.fontSize.large,
+    fontWeight: 'bold',
+    color: palette.neutral
+  },
+  userBadges: {
+    height: 22
```

Comment:
Should this be set explicitly? Also it looks like figma has these as 14x14?

Reply:
Huh... I had that as a hacky solution to have them vertically centered.. but the issue seems to have gone away now. Removing this. And they default to 14x14.